rust/release/src/publisher.rs

```diff
@@ -442,6 +442,66 @@ fn mk_blob_tuples(release: &Release, hash: &Oid, os_arch: OsArch) -> Vec<(Source
         (Product::TsSdk, _) => {}
         (Product::RustSdk, _) => {}
         (Product::CSharpSdk, _) => {}
+        (Product::AxCore, _) => {}
+        (Product::Ax, OS::android) => {
+            out.push((
+                format!("{}-binaries/actyx.apk", os_arch.os,),
```

Comment:
It just crossed my mind the question: With android what does `ax`-like app do?

Reply:
A "true" `ax`: panic
The `actyx.apk` is a bundled ax-core pretty much